Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -66,6 +66,18 @@ enum HappinessState
 	[SerializeField] float _emoteRoutineWait = 0f;
 	Coroutine _currentEmoteRoutine = null;
 
+	[Header( "Debug Settings" )]
+	[SerializeField] bool _disableStatDecrease = false;
```

Comment:
Debug settings should be at the bottom. As it stands `particlesRenderer` and `_deadParticleSystem` are showing up under the "Debug Settings" header. Also, make sure you do `[Space( 10 )]` when you do `[Header()]`.